src/sentry/api/endpoints/relay/project_configs.py

```diff
@@ -59,6 +65,57 @@ def _post(self, request: Request):
         else:
             return Response("Unsupported version, we only support versions 1 to 3.", 400)
 
+    def _post_or_schedule_by_key(self, request: Request):
+        request.relay_request_data.get("publicKeys")
+        public_keys = set(request.relay_request_data.get("publicKeys") or ())
+
+        configs = {}
+        pending = []
+        for key in public_keys:
+            computed = self._compute_proj_config(key)
+            if not computed:
+                pending.append(key)
+            else:
+                configs[key] = computed
+
+        if len(pending) > 0:
+            configs["pending"] = pending
+
+        return Response(configs, status=200)
+
+    def _compute_proj_config(self, public_key):
+        """
+        Returns the config of a project if it's in the cache; else, schedules a
+        task to compute and write it into the cache, and returns False.
+
+        Debouncing of the project happens after the task has been scheduled.
+        """
+        cached_config = projectconfig_cache.get(public_key)
+        if cached_config:
+            return cached_config
+        if projectconfig_debounce_cache.is_debounced(
+            public_key=public_key, project_id=None, organization_id=None
+        ):
+            return False
+
+        update_config_cache.delay(
```

Comment:
This task fails with errors internally if the specified public key does not exist. 
- Either we check for its existence before scheduling the task. That, however, would create more database queries in the endpoint, which we'd like to avoid to speed up its response.
- Alternatively, we change the task so that it doesn't fail with an error. 

Especially in the second case, we need to ensure that a cache entry is created so that we don't repeatedly create tasks. That cache entry should probably look like this:
https://github.com/getsentry/sentry/blob/918b6061381c2b139e2bce90c2177deced185096/src/sentry/tasks/relay.py#L84